tests/integration_tests/test_sim.py

```diff
@@ -116,6 +118,17 @@ def test_pin_mode(self, qtbot, sim_app, device_type):
         assert hw.get_pin_mode(17) == PinModes.INPUT
         assert hw.get_pin_mode(34) == PinModes.OUTPUT
 
+    def test_voltage_reading(self, qtbot, sim_app, device_type):
+        voltage_sensor = VoltageSensor(36)
```

Comment:
Instead of constructing this sensor here, it should be constructed in the profiles in `profiles/rockets` along with all the other sensors.